app/src/main/java/com/github/sdp/tarjetakuna/database/DBMagicCard.kt

```diff
@@ -0,0 +1,37 @@
+package com.github.sdp.tarjetakuna.database
+
+import androidx.room.Entity
+import com.github.sdp.tarjetakuna.model.MagicCard
+import com.google.gson.Gson
+
+/**
+ * Class to represent a MagicCard in the Firebase database
+ */
+@Entity(tableName = "magic_card", primaryKeys = ["code", "number"])

```

Comment:
Is there a particular reason not to use the API cardID instead of a mix of code and number (which can introduce errors on some particular cards)?